Content.Server/Administration/Systems/AdminSystem.cs

```diff
@@ -270,6 +270,12 @@ private PlayerInfo GetPlayerInfo(SessionData data, ICommonSession? session)
             overallPlaytime = playTime;
         }
 
+        // Moffstation - Start - Get watchlist status
+        var watchListed = false;
+        if (session != null)    // Second check so we don't modify upstream code
+            watchListed = _watchList.GetWatchListed(session);
```

Comment:
```suggestion
        // Second check so we don't modify upstream code
        var watchListed = session != null ? _watchList.GetWatchListed(session) : false;
```
I just personally hate
```csharp
var something = initializer;
if (someCondition)
    something = someOtherValue;
```
, coming from languages where local bindings are always immutable.